File: single_number.py

class Solution:
    # @param A, a list of integer
    # @return an integer
    def singleNumber(self, A):
        apro = {}
        v = 0
        ss = 0
        ss2 =0
        for it in A:
            ss += it
            if it not in apro:
                apro[it] = v
                v+=1
                ss2 += it
        return ss2*2-ss
            
        

File: singlenumber.cpp
class Solution {
public:
    int singleNumber(vector<int>& nums) {
        int ans = 0;
        for(int i=0;i<nums.size();i++)
        {
            ans = ans^nums[i];
        }
        return ans;
    }
};

// I have added a CPP solution for Single Number Leetcode 136
Kindly Acknowledge.
